aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,360 @@
+| How you are doing the writing	| Who they hear                               |
+| Style		                      | How your message comes across               |
+| What makes us uniquely us		  | How we communicate to our specific audience |
+
+### Voice
+
+Voice is all about who the reader hears talking in our messaging; it has to do with the worldview of the author. Voice can be spoken, witty, friendly, formal, etc. We can be perceived as being intelligent, experienced, mature, accepting, objective, etc.
+
+A person's voice typically does not change once established. If our voice (HPE's voice) is inconsistent, people will notice. For example, people notice the difference between writing academically when compared to sounding like a 5-year old. This kind of inconsistency can result in lack of trust.
+
+### Tone
+
+Tone has to do with what is conveyed or how it comes across. It can express seriousness, urgency, playfulness, objectivity, intimacy, condescension, etc.
+
+Unlike voice, tone can change based on the target audience and the subject matter. It is situational. For example, tone is very different when giving directions during an emergency vs ordering a pizza.
+
+### Point of View
+
+Point of view has to do with the perspective from which the message is written. First person perspective uses words like: I, me, us, we, our, etc. Second person uses any form of the word "you" which has the effect of addressing the reader. Third person directs the reader to things that are neither the writer or the reader.
+
+Examples:
+
+- *We noticed the server is malfunctioning.* (1st person-ish)
+- *Your server is malfunctioning.* (2nd person-ish)
+- *The server is malfunctioning.* (3rd person - no focus on the writer or the reader)
+
+### Our Voice and Tone
+
+We want our messaging to be informative, concise, and trustworthy. We should be perceived as intelligent, experienced, inclusive and non-threatening. Our messaging should be active, positive, objective, conversational American English written in a third person point of view.
+
+The following are some high-level guidelines for constructing good messaging:
+
+- Conversational American English
+- Correct grammar and punctuation
+- Active tense
+- Positive, not negative
+- Not robotic sounding
+- Never condescending
+- Establish context
+- Lead with what is most important
+- Inspire trust and confidence
+- Explain using terminology consistent with the UI
+- Avoid charged words like "*error*"
+- Avoid blame casting (on user or on software)
+- State facts without coloring them in opinions
+- Avoid personal pronouns
+- Careful with words like "*Please*"
+- Do not use "*contact your administrator*" (the admin is likely the user) or "*contact HPE*" (HPE might not be the support contact -- it might be a channel partner)
+- Read your message out loud to another person. Is this how you would say it out loud? Does the other person feel this is how two people would converse?
```

Comment:
Played with organizing by theme + supporting examples to help break up the content into more digestible chunks.

```suggestion
- Use HPE's voice and tone
    - Conversational American English
    - Correct grammar and punctuation
    - Active tense
    - Human and approachable, not robotic sounding
    - Positive, "yes we can," not negative
    - Never condescending
- Respect the user and their time
    - Establish context
    - Lead with what is most important
    - Explain using terminology consistent with the UI
    - Point the user directly to a resolution. Do not use "*contact your 
    administrator*" or "*contact HPE*" &mdash; the administrator is likely 
    the user and the support contact could be a channel partner. Instead, 
    provide a direct means to connect the user with the party able to resolve 
    the issue.
- Inspire trust and confidence
    - Use informative, concise language
    - State facts without coloring them in opinions
    - Avoid charged words such as "*error*"
    - Focus on the solution, avoid casting blame on the user or software
    - Focus on what is needed to be accomplished, avoid personal pronouns 
    (e.g. I, you, he, she, it, we, they, me, him, her, us, them)
- Ensure the voice is human and humane
    - Read your message out loud to another person. Is this how you would say it out loud? Does the other person feel this is how two people would converse?
```